releases/UWP/net-native2.2/README.md

```diff
@@ -2,8 +2,13 @@
 
 You can see what was included in each .NET Native 2.2 ([Microsoft.NETCore.UniversalWindowsPlatform 6.2.x](https://www.nuget.org/packages/Microsoft.NETCore.UniversalWindowsPlatform)) release below.
 
-When using Visual Studio these packages require Visual Studio 2017 Version 15.7 or later.
+When using Visual Studio these packages require Visual Studio 2017 Version 15.9 Preview 2 or later.
 
-### UWP 6.2.0 Preview (May 7th, 2018) 
+### UWP 6.2.0-preview1-26831-02 Preview (September 11th, 2018) 
+These are preview bits in advance of the release in November 2018
```

Comment:
Should we link to the 15.9 release site?  Also I'm not sure if they publish the release date.